packages/terra-switch/src/terra-dev-site/doc/example/DisabledSwitch.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Switch from 'terra-switch';
+
+export default () => (
```

Comment:
Nit: For consistency I think should export a named value in these examples. This is subject to personal preference, but I think the examples should be written in the same way we'd expect more complicated components to be written. 

```jsx
const DisabledSwitch = () => (
    <Switch
      id="disabledSwitch"
      labelId="label2"
      checked
      label="Label"
      disabled
    />
);

export default DisabledSwitch;
```